pyccl/errors.py

```diff
@@ -40,8 +84,8 @@ def __hash__(self):
 
     @classmethod
     def enable(cls):
```

Comment:
Are the deprecation warnings covered by the verbosity filter as well?

Reply:
I don't think so, but let me check

Reply:
No they are not. Do you think they should?

Reply:
My gut feeling is yes - they're good to know while building a model but you probably don't want them in an MCMC log.

Reply:
If they mute them and CCL's deprecation cycle completes, their MCMC will simply stop working, and they won't be able to figure out why. I don't think it's a good idea muting those.

Reply:
Well, if the deprecation cycle completes, they will get an error, which they should see, righ?

I'll mute those too @tilmantroester

Reply:
Indeed, but deprecation warnings have to do with CCL functionality whereas CCLWarnings mostly have to do with numerical stability, so they represent different things. At the very least I'd add a different flag to separate the verbosity level for deprecation-specific warnings. BTW now that v3 is out, is `CCLDeprecationWarning` used at all?

Reply:
CCLDeprecationWarnings are not used right now. I'll leave them there for the future in any case.

Reply:
They seemed to be used here:
https://github.com/LSSTDESC/CCL/blob/master/pyccl/baryons/baccoemu_baryons.py#L221
and the decorators in
https://github.com/LSSTDESC/CCL/blob/master/pyccl/_core/deprecations.py
But the latter don't seem to be used.

Reply:
As developers we can decide what the priority of the different warnings are. If we decide that specific deprecation warnings are sufficiently important, we can just tag them as "high", and they'll go through the filter.

@tilmantroester I've now added the CCLDeprecationWarnings to the list of warnings that can be suppressed if desired.